### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by underthehai**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -62,7 +62,15 @@ It is optimized for CLI users so that destinations and details can be added fast
...
-## Setting up [Coming soon]
+## Quick Start 
+
+1. Ensure you have Java `11` or above installed in your computer.
+2. Download the latest `Wanderlust.jar` [here](https://github.com/AY2021S1-CS2103-T14-3/tp/releases/tag/v1.2.1)
+3. Copy the file to the folder you want to use as the _home folder_ for Wanderlust
+4. Double click to start the app.
+5. Type the command in the command box and press Enter to execute. You can find the list of commands available in the [Command Summary](#command-summary)
+
+![Ui](images/userguidess/homescreen.png)
```
Comment (jeannetoh99) [inline docs/UserGuide.md]: I think userguidess is misspelt

File: docs/UserGuide.md [hunk 2]
```diff
@@ -72,13 +80,29 @@ The table briefly describes the commands and its usage. Full details will be giv
...
+Wishlist | TravelPlan
+----------  | ---------
+`goto -OBJECT KEYWORD`|`goto -OBJECT KEYWORD`
+`show -activity`| `show -OBJECT`
+`sort -activity` | `sort -OBJECT`
+`find -activity`| `find -OBJECT`
+`add -activity` |  `add -OBJECT`
+`delete -activity`| `delete -OBJECT`
+`edit -activity` | `edit -OBJECT`
+
```
Comment (jeannetoh99) [inline docs/UserGuide.md:103]: include move and copy under wishlist, and exit and help under both.

Comment (lyeyixian) [inline docs/UserGuide.md:105]: copy and move command

File: docs/UserGuide.md [hunk 3]
```diff
@@ -49,36 +50,67 @@ It is optimized for CLI users so that destinations and details can be added fast
...
+        2. [Sorting by date (L)](#2-sorting-by-date-l)
+        3. [Sorting by importance (L)](#3-sorting-by-importance-l)
+        4. [Sorting by name (L)](#4-sorting-by-name-l)
     * [Copy](#copy)
         1. [Copying an activity (L)](#1-copying-an-activity-l)
     * [Move](#move)
         1. [Moving an activity (L)](#1-moving-an-activity-l)
-* [FAQ \[Coming soon\]](#faq-coming-soon)
+    * [Help](#help)
+        1. [Refer to UserGuid (G)](#1-refer-to-userguide-g)
```
Comment (lyeyixian) [inline docs/UserGuide.md:62]: "UserGuide"

File: docs/UserGuide.md [hunk 4]
```diff
@@ -396,8 +460,23 @@ Format: `move ACTIVITY_INDEX TRAVELPLAN_INDEX`
 
 Example: `move 2 1`
 
+##Help
+
+### 1. Refer to UserGuide (G)
+
+A pop out button will provide a link to access Wanderlust UserGuide for help
```
Comment (lyeyixian) [inline docs/UserGuide.md:467]: "A pop-up window will provide the link to access Wanderlust UserGuide for help"

**Review by lyeyixian — APPROVED** [review]
LGTM

**Pull request merged**